### PR TITLE
Message status ux improvements

### LINKED ISSUE #4738: Read status UX improvements (follow up)
See open topics in https://github.com/nextcloud/spreed/pull/4231

- [x] Beautiful UI by @ma12-co
- [ ] hide indicator when hovered ? https://github.com/nextcloud/spreed/pull/4231#issuecomment-742522925
- [x] tooltip + aria labels for indicator ? (but might not be accessible when hovering the row, screen readers can likely reach it in browse mode)
- [ ] anything else ?

### PR DESCRIPTION
**Pull request by ma12-co**
fix #4738 

Peek 2020-12-10 17-16

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

**Comment by nickvergessen** [comment]
Peek 2020-12-11 11-41

Something weird is going on

**Comment by PVince81** [comment]
Looks fine in regular mode visually.

However with the Dyslexia font the timestamp wraps, and it messes up with the mouseenter/mouseleave event so the hover is broken. See the video above.

File: src/components/MessagesList/MessagesGroup/Message/Message.vue [hunk 1]
```diff
@@ -495,4 +522,13 @@ export default {
 	50% { background-color: var(--color-background-hover); }
 	100% { background-color: rgba(var(--color-background-hover), 0); }
 }
+
+.message-status {
+	margin: -8px 0;
+	width: 44px;
```
Comment (PVince81) [inline src/components/MessagesList/MessagesGroup/Message/Message.vue:528]: I think you can use $clickable-area here

**Review by PVince81 — CHANGES_REQUESTED** [review]
See comments

**Comment by ma12-co** [comment]
> However with the Dyslexia font the timestamp wraps, and it messes up with the mouseenter/mouseleave event so the hover is broken. See the video above.

I tried to reproduce but I'm unable to make it wrap

**Comment by nickvergessen** [comment]
English US locale so you have 11:50 AM (need 4 digits + 2 letters)

**Comment by ma12-co** [comment]
Updated the layout so that now the date, however long, never wraps.

Screenshot from 2020-12-16 18-28-06

File: src/components/MessagesList/MessagesGroup/Message/Message.vue [hunk 2]
```diff
@@ -448,26 +474,31 @@ export default {
...
+				}
 			}
 			& h6 {
 				margin-left: auto;
 			}
 		}
-	}
-	.icon-loading-small {
-		position: unset;
+	.date {
```
Comment (PVince81) [inline src/components/MessagesList/MessagesGroup/Message/Message.vue:499]:
```
ERROR in 
src/components/MessagesList/MessagesGroup/Message/Message.vue
 499:2  ✖  Expected indentation of 2 tabs   indentation
 500:3  ✖  Expected indentation of 3 tabs   indentation
```

File: src/components/MessagesList/MessagesGroup/Message/Message.vue [hunk 3]
```diff
@@ -57,21 +57,33 @@ the main body of the message as well as a quote.
...
-					v-if="showCommonReadIcon"
-					decorative
-					title=""
-					:size="16" />
-				<Check
-					v-else-if="showSentIcon"
-					decorative
-					title=""
-					:size="16" />
+				<span v-if="hasDate" v-tooltip.auto="messageDate" class="date">{{ 'looongdateee' }}</span>
```
Comment (PVince81) [inline src/components/MessagesList/MessagesGroup/Message/Message.vue:60]: some leftover from testing ?

**Review by PVince81 — COMMENTED** [review]
I guess this wasn't ready for re-review yet. Let us know when it is

**Comment by ma12-co** [comment]
> I guess this wasn't ready for re-review yet. Let us know when it is


Uff, sorry.. I amended my commit without first saving the last changes :facepalm: : ...Ready for review now

**Comment by nickvergessen** [comment]
Messages of others look out of place, I'm Alice:
Bildschirmfoto von 2020-12-17 10-13-50

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**